05-extra.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Additional Topics
```

Comment:
The `minutes` field is unique to lesson pages: see [LAYOUT.md](https://github.com/swcarpentry/lesson-template/blob/gh-pages/LAYOUT.md) for details.

I don't think the template spec takes into account whether or not a given Topic page is optional. It should be present on all topic pages (named as ##-word.md).